apps/dashboard/src/components/side-navigation/free-trial-card.tsx

```diff
@@ -0,0 +1,74 @@
+import { useBillingSubscription } from '@/hooks';
+import { LogoCircle } from '../icons';
+import { RiArrowRightDoubleLine, RiInformationFill } from 'react-icons/ri';
+import { Progress } from '../primitives/progress';
+import { Button } from '../primitives/button';
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger, TooltipArrow } from '../primitives/tooltip';
+import { LEGACY_ROUTES } from '@/utils/routes';
+
+const transition = 'transition-all duration-300 ease-out';
+
+const pluralizeDaysLeft = (numberOfDays: number) => {
+  return `${numberOfDays} day${numberOfDays > 1 ? 's' : ''}`;
+};
+
+export const FreeTrialCard = () => {
+  const { subscription, isLoading } = useBillingSubscription();
+  const { trial, hasPaymentMethod } = subscription;
+
+  if (isLoading || !trial.isActive || hasPaymentMethod) {
+    return null;
+  }
+
+  const pluralizedDays = pluralizeDaysLeft(trial.daysLeft);
+
+  return (
+    <a
+      href={LEGACY_ROUTES.BILLING}
+      className="bg-background group absolute bottom-3 left-2 flex w-[calc(100%-1rem)] cursor-pointer flex-col gap-2 rounded-lg p-3 shadow"
+    >
+      <div className="flex items-center gap-1.5">
+        <div
+          className={`bg-primary flex h-4 w-4 items-center justify-center rounded-full ${transition} group-hover:bg-transparent`}
+        >
+          <LogoCircle className={`h-3 w-3 ${transition} group-hover:h-4 group-hover:w-4`} />
+        </div>
+        <span className="text-foreground text-sm">{pluralizedDays} left on trial</span>
+        <TooltipProvider>
+          <Tooltip>
+            <TooltipTrigger className="ml-auto">
+              <span className="relative flex size-4 items-center justify-center">
+                <RiArrowRightDoubleLine
+                  className={`text-foreground/40 size-4 opacity-100 ${transition} group-hover:opacity-0`}
+                />
+                <RiInformationFill
+                  className={`text-foreground/40 absolute left-0 top-0 size-4 opacity-0 ${transition} group-hover:opacity-100`}
+                />
+              </span>
+            </TooltipTrigger>
+            <TooltipContent variant="light" size="lg" side="right" className="w-48">
+              <TooltipArrow variant="light" className="-translate-y-[1px]" />
```

Comment:
one thing that I also don't like about the arrow is this translate based on the content side, I tried to do that with classes but failed :( please suggest something better

Reply:
What do you mean by `content side`?

Reply:
🤦